docs/knowledge_base/policy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Policy
+category: 6255e67693d5200013b1fa41
+hidden: false
+---
+
+# Overview
+Phylum's custom policy support allows you to take control over the allow/block decision for dependencies being added to a Phylum project.
+
+# How it works
+
+When a developer introduces dependency changes, either in a pull request when using one of the source control server integrations or when using Phylum's CLI extensions, a simple policy is applied to determine whether or not that change should be allowed. This policy is implemented using [Open Policy Agent] and may be overridden in project preferences.
+
+Suppressed issues and ignored dependencies when performing a comparison are filtered out before applying the policy and will not be visible in the policy input.
```

Comment:
This paragraph feels out of place. The section is talking about how we evaluate policies and the previous paragraph says that we use OPA and the user can provide their own, but then we start talking about how suppressed issues and ignored dependencies affect policy input but all three of those things are first introduced in this paragraph.

Reply:
I removed a few things, but will probably have to come back to this section after I interact with more users.